interface/modules/custom_modules/oe-module-faxsms/openemr.bootstrap.php

```diff
@@ -15,13 +15,15 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+require_once(__DIR__ . "/../../../globals.php");
```

Comment:
this doesn't seem right. don't think any of the other module bootstrap's need this

Reply:
Testing global scopes depth trying to figure out why most other modules are using `globals $GLOBALS` in class methods!
A php no no imo. I've mention this on several occasions as it shouldn't be needed ever and if it is, then something is wrong upstream! One of those "drives me crazy"!
Thanks, I forgot to remove and you're right that globals should be in scope here....